packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -186,6 +187,19 @@ class BondWithdrawalWatcher extends BaseWatcher {
         throw new BonderFeeTooLowError(msg)
       }
 
+      const l1Bridge = this.getSiblingWatcherByChainSlug(Chain.Ethereum).bridge as L1Bridge
+      const sourceChainTotalHTokens = await l1Bridge.getChainBalance(sourceChainId)
+      const pendingAmount = await sourceL2Bridge.getPendingAmountForChainId(destinationChainId)
+      const hTokensAvailable = sourceChainTotalHTokens.add(pendingAmount)
+      const sourceChain = this.chainIdToSlug(sourceChainId)
+      const isFromNonRollup = [Chain.Polygon, Chain.Gnosis].includes(sourceChain)
```

Comment:
This should apply to every chain, so the concept of `isFromNonRollup` is unnecessary here.